app/Http/Backoffice/Handlers/Auth/AuthForgotPasswordHandler.php

```diff
@@ -21,8 +22,9 @@ public function __invoke(Redirector $redirector, ForgotPasswordRequest $request,
     {
         $email = $request->getEmail();
 
-        /** @var \Digbang\Security\Users\User $user */
-        if (! $email || ! ($user = $securityApi->users()->findByCredentials(['email' => $email]))) {
+        /** @var User|null $user */
+        $user = $securityApi->users()->findByCredentials(['email' => $email]);
+        if (! $email || ! $user) {
```

Comment:
Esto no hace lo mismo que el codigo anterior. Al chequear primero por mail, evitas un pedido al pedo a la base. Ademas, podria no gustarle al find que le mandes ese parametro en null.

Reply:
Tienes razon... Pero asi como estaba antes me da este error :(

`
 ------ ----------------------------------------------------------------- 
  Line   app/Http/Backoffice/Handlers/Auth/AuthForgotPasswordHandler.php  
 ------ ----------------------------------------------------------------- 
  26     Variable $user in PHPDoc tag @var does not exist.                  
 ------ ------------------------------------------------------------
`

Reply:
No lo hagas como estaba antes. Mejora ambas cosas...
Podrias hacer un if para el email y otro para el user. Prefiero repetir el codigo de la redireccion a hacer un pedido de mas a la base de datos.

Reply:
Igual ahi lo arregle. Lo deje como estaba antes con un solo pedido.

Reply:
Mentira, no lo arregle, pero no tiene sentido esta validacion ya que el Request valida, asi que chao.